config.js

```diff
@@ -512,6 +512,22 @@ var config = {
     // If set to true all muting operations of remote participants will be disabled.
     // disableRemoteMute: true,
 
+    /**
+     External API url used to receive branding specific information.
+     If there is no url set or there are missing fields, the defaults are applied.
+     None of the fieds are mandatory and the response must have the shape:
+     {
+         // The hex value for the colour used as background
+         backgroundColor: '#fff',
+         // The url for the image used as background
+         backgroundImageUrl: 'https://example.com/background-img.png',
+         // The anchor url used when clicking the logo image
+         logoClickUrl: 'https://example-company.org',
+         // The url used for the image used as logo
+         logoImageUrl: 'https://example.com/logo-img.png'
+    */
```

Comment:
nit: close the object literal

Reply:
Missed that. Thanks!

Reply:
Hey, I think you are also missing a key name for this object? Or are these values (`logoClickUrl `, `logoImageUrl`, etc) supposed to be at the top level?